toolchest_client/tools/star.py

```diff
@@ -30,9 +30,11 @@ def __init__(self, tool_args, output_name, inputs, input_prefix_mapping,
             max_inputs=2,
             database_name=database_name,
             database_version=database_version,
-            parallel_enabled=True,
+            parallel_enabled=True if parallelize else False,
```

Comment:
Thoughts on `parallel_enabled=parallelize`?

Reply:
That works too! In the back of my mind I was thinking of the user input as dirty and potentially non-boolean given that we don't have types implemented.

I have a slight preference for this, but am open to changing if you have a preference.

Reply:
Merging in the meantime as this would be a trivial fix and I want to get the integration tests started